src/snowflake/connector/result_batch.py

```diff
@@ -123,6 +124,7 @@ def create_batches_from_response(
                         compressedSize=c["compressedSize"],
                     ),
                     column_names,
+                    schema,
                     column_converters,
```

Comment:
Removed `column_names` as that can easily be extracted from the schema. `column_converters` still preserved since they need more than what the schema contains.